astronomer/providers/databricks/operators/notebook.py

```diff
@@ -0,0 +1,184 @@
+                notebook_params=[],
+            )
+            with task_group:
+                notebook_1 = DatabricksNotebookOperator(
+                    task_id="notebook_1",
+                    databricks_conn_id="databricks_conn",
+                    notebook_path="/Users/daniel@astronomer.io/Test workflow",
+                    source="WORKSPACE",
+                    job_cluster_key="Shared_job_cluster",
+                )
+                notebook_2 = DatabricksNotebookOperator(
+                    task_id="notebook_2",
+                    databricks_conn_id="databricks_conn",
+                    notebook_path="/Users/daniel@astronomer.io/Test workflow",
+                    source="WORKSPACE",
+                    job_cluster_key="Shared_job_cluster",
+                    notebook_params={
+                        "foo": "bar",
+                    },
+                )
+                notebook_1 >> notebook_2
+
+
+        :param notebook_path: the path to the notebook in Databricks
+        :param source: Optional location type of the notebook. When set to WORKSPACE, the notebook will be retrieved
+        from the local Databricks workspace. When set to GIT, the notebook will be retrieved from a Git repository
+        defined in git_source. If the value is empty, the task will use GIT if git_source is defined
+        and WORKSPACE otherwise. For more information please visit
+        https://docs.databricks.com/dev-tools/api/latest/jobs.html#operation/JobsCreate
+        :param databricks_conn_id: the connection id to use to connect to Databricks
+        :param notebook_params: the parameters to pass to the notebook
+        :param kwargs:
+    """
+
+    template_fields = "databricks_run_id"
+
+    def __init__(
+        self,
+        notebook_path: str,
+        source: str,
+        databricks_conn_id: str,
+        notebook_params: dict | None = None,
+        **kwargs,
+    ):
+        """
+        Create a Databricks notebook operator.
+
+        :param notebook_path: the path to the notebook in Databricks
+        :param source: Optional location type of the notebook. When set to WORKSPACE, the notebook will be retrieved
+        from the local Databricks workspace. When set to GIT, the notebook will be retrieved from a Git repository
+        defined in git_source. If the value is empty, the task will use GIT if git_source is defined
+        and WORKSPACE otherwise. For more information please visit
+        https://docs.databricks.com/dev-tools/api/latest/jobs.html#operation/JobsCreate
+        :param databricks_conn_id: the connection id to use to connect to Databricks
+        :param notebook_params: the parameters to pass to the notebook
+        :param kwargs:
+        """
+        self.notebook_path = notebook_path
+        self.source = source
+        self.notebook_params = notebook_params or {}
+        self.databricks_conn_id = databricks_conn_id
+        self.databricks_run_id = ""
```

Comment:
can we have a databricks job link on the UI for the users to quickly see the job status from airflow, similar to how google bigquery operators have it?